src/__tests__/resources/timeSeriesList.unit.spec.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 Cognite AS
+import MockAdapter from 'axios-mock-adapter';
+import { uniqBy } from 'lodash';
+import { CogniteClient } from '../..';
+import { AssetList } from '../../resources/classes/assetList';
+import { TimeSeriesList } from '../../resources/classes/timeSeriesList';
+import { randomInt, setupLoggedInClient } from '../testUtils';
+
+describe('TimeSeriesList class unit test', async () => {
+  let axiosMock: MockAdapter;
+  let client: CogniteClient;
+  let newAsset: any;
+  let createdTimeSeries: TimeSeriesList;
+  let timeSeriesWithAssetId: any;
+  let createdAssets: AssetList;
+  let datapointArray: any[];
+  let assetsArray: any[] = [];
+  let timeseriesArray: any[] = [];
+  beforeAll(() => {
+    client = setupLoggedInClient();
+    axiosMock = new MockAdapter(client.instance);
+  });
+  beforeEach(async () => {
```

Comment:
Are you sure you want to use `beforeEach` and not `beforeAll` here?

Reply:
Moved to beforeAll